examples/cloudrun/analysis/.pipe.yaml

```diff
@@ -11,7 +11,18 @@ spec:
       # If this stage finds any not good metrics of the new version,
       # a rollback process to the previous version will be executed.
       - name: ANALYSIS
+        with:
+          duration: 10m
+          metrics:
+            - provider: prometheus-dev
+              interval: 2m
+              expected:
+                max: 0.01
+              query: |
+                sum by (job) (rate(http_requests_total{status=~"5.*", job="analysis-by-metrics"}[5m]))
```

Comment:
While I think it's not common to use Prometheus for monitoring CloudRun, for now it's fine until we support Cloud Monitoring as an analysis provider.

Reply:
Once got supported, I'm going to update here

Reply:
Right. Let me update this to use HTTP check.

Reply:
Fixed it!